src/persist-client/src/read.rs

```diff
@@ -109,6 +110,7 @@ where
     /// [differential_dataflow::consolidation::consolidate_updates].
     ///
     /// An None value is returned if this iterator is exhausted.
+    #[instrument(level = "debug", name = "snap::next", skip_all, fields(shard = %self.shard_id))]
```

Comment:
god I love tracing, this `shard =%self.shard_id` is great!

Reply:
does this `name` become the span name in otel data?

Reply:
> does this name become the span name in otel data?

yup! (open-loop doesn't use snapshots but here's the listen::next one)

<img width="341" alt="image" src="https://user-images.githubusercontent.com/52528/172483986-cda742c0-3396-47d5-84b8-5692800d1fa7.png">